lib/ProductOpener/Food.pm

```diff
@@ -5793,40 +5793,51 @@ This function was first designed to assign a CIQUAL category to products, based
 the mapping of the Open Food Facts categories to the French CIQUAL categories.
 
 It may be used for other properties in the future.
+
+agribalyse_food_code:en:42501
+agribalyse_proxy_food_code:en:43244
+
 =cut
 
-sub assign_category_properties_to_product($) {
+sub assign_categories_properties_to_product($) {
 
 	my $product_ref = shift;
 
-	$product_ref->{category_properties} = {};
+	$product_ref->{categories_properties} = {};
+	$product_ref->{categories_properties_tags} = [];
+
+	# Simple properties
 
-	foreach my $property ("ciqual_food_code:en:", "ciqual_food_name:en", "ciqual_food_name:fr") {
+	foreach my $property ("agribalyse_food_code:en", "agribalyse_proxy_food_code:en", "ciqual_food_code:en:") {
+
+		my $property_name = $property;
+		$property_name =~ s/:en$//;
 
 		# Find the first category with a defined value for the property
 
 		if (defined $product_ref->{categories_tags}) {
 			foreach my $categoryid (reverse @{$product_ref->{categories_tags}}) {
 				if ((defined $properties{categories}{$categoryid}) and (defined $properties{categories}{$categoryid}{$property})) {
-					$product_ref->{category_properties}{$property} = $properties{categories}{$categoryid}{$property};
+					$product_ref->{categories_properties}{$property} = $properties{categories}{$categoryid}{$property};
```

Comment:
Is this a breaking change? I've found this property set in existing products in the public API.

Reply:
It's something I added a couple weeks ago, it is a work in progress and subject to change, it's not something documented and it is not used by anyone.
I'll update all products to remove the field with the old name.